core/stores/mon/bulkinserter.go

```diff
@@ -42,6 +42,26 @@ func NewBulkInserter(coll *mongo.Collection, interval ...time.Duration) *BulkIns
 	}
 }
 
+// NewBatchInserter returns a BulkInserter.
+func NewBatchInserter(coll Collection, interval ...time.Duration) *BulkInserter {
+	// When cloning, there is no possibility of failure at the bottom layer, so the error is not handled
+	cloneColl, _ := coll.Clone()
```

Comment:
even the err is impossible to be not nil, we still need to check it. Because if the underly code changed to return a not nil error, we'll get unexpected behavior.